GrappleGirlGame/Music.lua

```diff
@@ -0,0 +1,52 @@
+--[[
+Music = {}
+Music.queue = {}
+
+function Music:play(data)
+    -- In love2d you can use Static or Stream for playing Music
+	local src = love.audio.newSource(data, "static")
+    src:setVolume(0.5)
+    src:play()
+end
+--]]
+do
```

Comment:
Why is this entire file in a do statement?

Reply:
Fixed